htdocs/main.inc.php

```diff
@@ -2138,6 +2153,17 @@ function top_menu_user($hideloginname = 0, $urllogout = '')
 	                $("#topmenu-login-dropdown").toggleClass("open");
 	            });
 
+	            $(".copyToClipboard").on("click", function() {
```

Comment:
To add feature for a copy/paste, there is already an implemented solution:
instead of 
print $myvalue
just do
print showValueWithClipboardCPButton($myvalue);

Reply:
Thanks @eldy 

It's done